### PR TITLE
SetUp for Routers

### PR DESCRIPTION
**Pull request by Naman-B-Parlecha**
will not change the card which u had mentioned yesterday

### CODE REVIEW

File: src/App.jsx
Comment (mohit-nagaraj) [inline src/App.jsx:1]: import './App.css'

**Review by mohit-nagaraj — COMMENTED** [review]
It works, so should be more than enough.

File: src/App.jsx
Comment (Naman-B-Parlecha) [reply src/App.jsx:1]: added semicolon was that what u meant or something else anna?

Comment (mohit-nagaraj) [reply src/App.jsx:1]: yup that one only

**Comment by mohit-nagaraj** [comment]
Please do resolve the conflicts(not possible at my end) and merge it off
image

**Comment by mohit-nagaraj** [comment]
Ay bro why did you close the pr & also remove the branch? i am assuming u do have changes on your local repo. Btw u adding the updated files? and wht to add for manjesh anna?

**Comment by Naman-B-Parlecha** [comment]
I was not able to update the dependency on this branch dk why was getting conflict and when searching for them 
Not able to find them even after changing was getting conflicts 
So made a new branch and upload all updated thing there

Any issue or something?
Please let me know if I made some mistake
Will not do again🙌

**Pull request closed without merging**